flow360/component/simulation/services.py

```diff
@@ -767,3 +772,54 @@ def update_simulation_json(*, params_as_dict: dict, target_python_api_version: s
         # Expected exceptions
         errors.append(str(e))
     return updated_params_as_dict, errors
+
+
+def validate_expression(variables: list[dict], expressions: list[str]):
+    """
+    Validate an expression using the specified variable space
```

Comment:
```suggestion
    Validate all given expressions using the specified variable space
```
If I understand correctly?

Reply:
Fixed in [6db9e09](https://github.com/flexcompute/Flow360/pull/946/commits/6db9e0930c2bd723625a8df6e07b602aa577a12a)